src/main/java/io/github/genomicdatainfrastructure/daam/api/ApplicationCommandApiImpl.java

```diff
@@ -114,4 +105,16 @@ private String userId() {
         var principal = (OidcJwtCallerPrincipal) identity.getPrincipal();
         return principal.getClaim(USER_ID_CLAIM);
     }
+
+    @Override
+    public Response acceptApplicationTermsV1(Long id, AcceptTermsCommand acceptTermsCommand) {
+        String userId = userId();
+
+        io.github.genomicdatainfrastructure.daam.remote.rems.model.AcceptLicensesCommand remoteAcceptLicensesCommand = new io.github.genomicdatainfrastructure.daam.remote.rems.model.AcceptLicensesCommand();
```

Comment:
important: you should use `AcceptTermsCommand` instead, and pass it to the service `acceptTermsService` directly.

Internally you parse `AcceptTermsCommand` into `AcceptLicensesCommand`.

Reply:
Changed it, can you check?